_includes/manuals/1.21/1.2_release_notes.md

```diff
@@ -4,9 +4,293 @@ This section will be updated prior to the next release.
+* Fix Rubocop: Layout/RescueEnsureAlignment ([#25325](https://projects.theforeman.org/issues/25325))
+* Fix Rubocop: Performance/UnneededSort ([#25324](https://projects.theforeman.org/issues/25324))
+* Update Rubocop to 0.59.2 ([#25300](https://projects.theforeman.org/issues/25300))
+* Invalid Host header while using webpack in dev mode ([#25252](https://projects.theforeman.org/issues/25252))
+* webpack's i18n is missing in Storybook documentation  ([#25218](https://projects.theforeman.org/issues/25218))
+* Create multiexception class for better exception handling in core. ([#24193](https://projects.theforeman.org/issues/24193))
+* Fix Rubocop: Style/PreferredHashMethods ([#19891](https://projects.theforeman.org/issues/19891))
+* Fix Rubocop: Style/For ([#19860](https://projects.theforeman.org/issues/19860))
+* Fix Rubocop: Metrics/PerceivedComplexity ([#19818](https://projects.theforeman.org/issues/19818))
+* Fix Rubocop: Metrics/ParameterLists ([#19817](https://projects.theforeman.org/issues/19817))
+* Fix Rubocop: Metrics/ModuleLength ([#19816](https://projects.theforeman.org/issues/19816))
+* Fix Rubocop: Metrics/MethodLength ([#19815](https://projects.theforeman.org/issues/19815))
+* Fix Rubocop: Metrics/LineLength ([#19814](https://projects.theforeman.org/issues/19814))
+* Fix Rubocop: Metrics/CyclomaticComplexity ([#19813](https://projects.theforeman.org/issues/19813))
+* Fix Rubocop: Metrics/ClassLength ([#19812](https://projects.theforeman.org/issues/19812))
+* Fix Rubocop: Metrics/BlockNesting ([#19811](https://projects.theforeman.org/issues/19811))
+* Fix Rubocop: Metrics/BlockLength ([#19810](https://projects.theforeman.org/issues/19810))
+* Fix Rubocop: Metrics/AbcSize ([#19809](https://projects.theforeman.org/issues/19809))
+* Fix Rubocop: Performance/StringReplacement ([#25370](https://projects.theforeman.org/issues/25370))
+* Update Rubocop to 0.50 ([#25249](https://projects.theforeman.org/issues/25249))
+
+#### Facts
+* Custom boolean facts with false value are not managed by Satellite. ([#25483](https://projects.theforeman.org/issues/25483))
+
+#### Foreman modules
+* Avoid storing puppetserver version related parameters in the answer file ([#25839](https://projects.theforeman.org/issues/25839))
+* make taxonomy and login settings optional ([#25787](https://projects.theforeman.org/issues/25787))
+
+#### Hammer CLI
+* Update api doc used in foreman tests to match Foreman 1.20 ([#25376](https://projects.theforeman.org/issues/25376))
+* Report templates command ([#24489](https://projects.theforeman.org/issues/24489))
+* hammer template dump does not request password when output redirected to a file ([#14832](https://projects.theforeman.org/issues/14832))
+
+#### Hammer core
+* Add switch to hammer CLI to disable the defaults ([#25307](https://projects.theforeman.org/issues/25307))
+* Allow modification of output fields ([#21634](https://projects.theforeman.org/issues/21634))
+
```

Comment:
Not sure what should be included in the release notes but this is the full list. I'll update the targets in redmine
```md
* Fixed instructions in i18n docs ([#25724](http://projects.theforeman.org/issues/25724))
* Option validators can be mixed with sources ([#22253](http://projects.theforeman.org/issues/22253))
* Validator uses options from option collector ([#19574](http://projects.theforeman.org/issues/19574))
* hammer -h does work with --output json, csv, yaml ([#25438](http://projects.theforeman.org/issues/25438))
* Update rpm names to use tfm ([PR #296](https://github.com/theforeman/hammer-cli/pull/296))
* Allows modification of output fields ([PR #275](https://github.com/theforeman/hammer-cli/pull/275)) ([#21634](http://projects.theforeman.org/issues/21634))
* Allow skipping message tests ([PR #292](https://github.com/theforeman/hammer-cli/pull/292)) ([#7451](http://projects.theforeman.org/issues/7451))
* Helper for editing text in editor ([PR #294](https://github.com/theforeman/hammer-cli/pull/294)) ([#24489](http://projects.theforeman.org/issues/24489))
* Add ability to disable defaults ([PR #293](https://github.com/theforeman/hammer-cli/pull/293)) ([#25307](http://projects.theforeman.org/issues/25307))
* Allow modification of additional help texts ([PR #277](https://github.com/theforeman/hammer-cli/pull/277)) ([#21633](http://projects.theforeman.org/issues/21633))
```